helm/druid/README.md

```diff
@@ -96,10 +96,16 @@ The following table lists the configurable parameters of the Druid chart and the
 | `configVars`                              | druid configuration variables for all components         | ``                                         |
 | `gCloudStorage.enabled`                  | look for secret to set google cloud credentials         | `false`                                    |
 | `gCloudStorage.secretName`               | secretName to be mounted as google cloud credentials    | `false`                                    |
+| `rbac.create`                            | Create roles and roleBindings for service Accounts      | `true`                                     |
 | `broker.enabled`                         | enable broker                                           | `true`                                     |
 | `broker.name`                            | broker component name                                   | `broker`                                   |
 | `broker.replicaCount`                    | broker node replicas (deployment)                       | `1`                                        |
 | `broker.port`                            | port of broker component                                | `8082`                                     |
+| `broker.serviceAccount.create`           | Create a service account for broker service             | `true`                                     |
+| `broker.serviceAccount.name`             | Service account name                                    | `true`                                     |
```

Comment:
The documentation looks off to me. Sorry I didn't catch this before. Looks like it says the default for everything is `true` but that's not actually the case.

Reply:
No apologies, this is my fault.  Sorry I missed it during pre-commit review. Fixed the documentation and merged upstream/master again.